src/main/resources/db/changelog/changes/08-create_cart_items_table.yaml

```diff
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_cart_items_table
+      author: Volodymyr Ozemblovskyi
+      changes:
+        - createTable:
+            tableName: cart_items
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
nullable is redundant here as you have primary key constraint, but that's a minor thing